### DISCUSSION TITLE
Incorrect Typst code highlighting

### DISCUSSION THREAD

**Discussion by cu1ch3n** [github-linguist/linguist#6789] [General]
The highlighting of the code below seems to be incorrect.
https://github.com/cu1ch3n/karenda/blob/631f68f70202c2142a4b042f2d4b0c6e71bad69e/lib.typ#L164-L190

![](https://github.com/github-linguist/linguist/assets/80438676/f62d4bd7-3872-431e-a7f2-cb7b2cc7a334)

While it is correctly highlighted in my VS Code:

![](https://github.com/github-linguist/linguist/assets/80438676/860ac7f0-4142-4873-8544-a5a4908f6251)

**Comment by lildude** [comment]
This will be a problem with the upstream grammar. As detailed in our [troubleshooting doc](https://github.com/github-linguist/linguist/blob/4ac734c15a96f9e16fd12330d0cb8de82274f700/docs/troubleshooting.md#theres-a-problem-with-the-syntax-highlighting-of-a-file), you'll need to raise an issue with the upstream grammar.

**Reply by cu1ch3n** [reply]
Thanks!

**Comment by cu1ch3n** [comment]
@michidk Hi! Thanks for contributing the grammar. Would you like to take a look at this issue? Thanks!
